app/src/main/java/ch/epfl/sdp/drone3d/ui/auth/RegisterActivity.kt

```diff
@@ -47,8 +47,20 @@ class RegisterActivity : AuthActivity() {
     }
 
     fun register(@Suppress("UNUSED_PARAMETER") view: View) {
-        startProcess(authService.register(emailEditText.text.toString(), passwordEditText.text.toString()),
+        val emailText = emailEditText.text.toString()
+        val passwordText = passwordEditText.text.toString()
+        if (emailText == "" || passwordText == "") {
+            ToastHandler.showToast(baseContext, R.string.login_fail)
+            infoText.text = getString(R.string.email_or_password_empty)
+            infoText.setTextColor(android.graphics.Color.RED)
+            infoText.visibility = View.VISIBLE
```

Comment:
Just a suggestion: as we use these same 3 lines in RegisterActivity, LoginActivity and AuthActivity, maybe create a function in AuthActivity that will display an error message